examples/highway_convergence_report.txt

```diff
@@ -0,0 +1,111 @@
+
```

Comment:
@vivekyadav26 why tracking this file?

Reply:
We don't have to track this file.

Reply:
Will you please add it to the gitignore?

Reply:
Added to gitignore.